lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -37,7 +37,11 @@ export default class ItinerarySummary extends Component {
           <div className='header'>{formatDuration(itinerary.duration)}</div>
 
           {/* Duration as time range */}
-          <div className='detail'>{formatTime(itinerary.startTime, timeOptions)} - {formatTime(itinerary.endTime, timeOptions)}</div>
+          <div className='detail'>
+            {formatTime(itinerary.startTime, timeOptions)}
+            -
+            {formatTime(itinerary.endTime, timeOptions)}
```

Comment:
Are the spaces surrounding the `-` preserved here properly?

Reply:
Good point, the spaces would not be preserved, will fix.